relay-metrics/src/lib.rs

```diff
@@ -68,13 +68,16 @@
     html_favicon_url = "https://raw.githubusercontent.com/getsentry/relay/master/artwork/relay-icon.png"
 )]
 
-mod aggregation;
+/// Only public for benchmarks.
+pub mod aggregator;
+
+mod aggregatorservice;
 mod bucket;
 mod protocol;
 mod router;
 mod statsd;
 
-pub use aggregation::*;
+pub use aggregatorservice::*;
```

Comment:
We can still re-export the parts of the `aggregator` module that are not supposed to be hidden:

```suggestion
pub use aggregator::AggregatorConfig;
pub use aggregatorservice::*;
```